src/atomate2/abinit/utils/common.py

```diff
@@ -35,6 +42,18 @@
 logger = logging.getLogger(__name__)
 
 
+class AbiConvergenceWarning:
```

Comment:
What is the purpose of this object? If you just need a list of the options you can define a constant with the list.
However, given how this is used (see other related comment), it can maybe be removed entirely?